### ISSUE TITLE
Tables within Tables

### ISSUE THREAD

**Issue by RichardNeill** [gajus/wholly#12]
If the table.highlightable contains another table, some really weird things occur. 

* The highlighting mechanism binds the x,y of the outer-table to the x,y of the inner table, so both sets of highlights trigger each other.
* If we're unlucky, the columns in the outer table can get a "kink" in which cells are highlighted.

There may be a better way, but here is my fix.

1.   Change wholly.js:
 
 var tableIndex = wholly.generateTableMatrix(table),
	    //If the table contains another table, some 'bad' things happen with the counters. 
           //Add .fixhighlightable to the tr  of the inner table.
	    //rows = table.find('tr');
	    rows = table.find('tr:not(.fixhighlightable)');

2. In the html, edit the inner table, and add .fixhighlightable on every &lt;tr&gt;. 

This fix would be neater if wholly could either scan for and ignore the inner table, or if the fixhighlightable class were applied to the inner table.

**Comment by gajus** [comment]
PR is welcome.

**Comment by RichardNeill** [comment]
Thanks. This is a rather ugly hack though - surely there's a better way?

**Comment by gajus** [comment]
I didn't mean PR this, but a PR that properly solves it.

**Comment by RichardNeill** [comment]
Yes, but sadly, that's a bit beyond me. I can't figure out how to write a selector that matches all rows of a given table, which are not themselves contained in another table. 
Consider:    &lt;table&gt; ... &lt;tr&gt;  ... &lt;td&gt; ....  &lt;table&gt;  ... &lt;tr&gt;
So what Jquery selector would match the first &lt;tr&gt; and not the second &lt;tr&gt;?

**Comment by gajus** [comment]
`table > tr` (and a variations of that that incorporate `thead` and `tbody`).

**Comment by RichardNeill** [comment]
Thanks. Here is the fix that I believe works correctly, and which I've tested. The aim of the fix is to ensure that wholly doesn't pay any attention to any table nested within the main table. Because if it did, then the counters go wrong in interesting ways, resulting in
*  simultaneous higlighting of (eg) the 2nd column in both tables,  
* kinks in the highlighting of the outer table.

The solution is to change the selector as follows.

wholly.indexTable = function (table) {
    var tableIndex = wholly.generateTableMatrix(table),
    //rows = table.find('tr');
    rows = table.find('> thead > tr, > tbody > tr, > tfoot > tr, > tr');
